static/webpack-config.js

```diff
@@ -143,42 +143,11 @@ module.exports = function () {
             }
           ],
         },
-        {
-          test: /\.html$/i,
-          use: [
-            {
-              loader: 'html-loader',
-              options: {
-                sources: {
-                  list: [
-                    {
-                      tag: 'link',
-                      attribute: 'href',
-                      type: 'src',
-                      filter: (tag, attribute, attributes) => {
-                        const isPreload = attributes.find(attr => {
-                          return attr.name === 'rel' && attr.value === 'preload';
-                        });
-                        const isFont = attributes.find(attr => {
-                          return attr.name === 'as' && attr.value === 'font';
-                        });
-                        return isPreload && isFont;
-                      }
-                    }
-                  ],
-                  urlFilter: (attribute, value) => {
-                    return /^[./]*static\/assets\//.test(value);
-                  }
-                },
-              }
-            }
-          ]
-        },
         {
           test: /\.(png|ico|gif|jpe?g|svg|webp|eot|otf|ttf|woff2?)$/,
```

Comment:
@oshi97 Can you confirm this is only impacting fonts based on this comment?: https://github.com/yext/answers-hitchhiker-theme/pull/889#discussion_r673379616

Reply:
I just want to confirm whether or not removing the hash will have impacts on the other files types listed in the test

Reply:
should be fine